### ISSUE TITLE
Make installation namespace and name available to actions

### ISSUE THREAD

**Issue by Jemgoss** [getporter/porter#2000]
**What design is being proposed?**
Currently, the template variable `installation.name` is set directly from the CNAB_INSTALLATION_NAME env variable. This is 
of the form `namespace/bundle_name` or just `/bundle_name` if no namespace is provided.

That form is not so useful as a substitution when writing action definitions, where it is much more helpful to be able to reference the namespace and bundle name separately. In particular, when defining a kubernetes action, it is clearly beneficial to be able to access the namespace component and also the bundle name can be used to generate an installation-specific resource name. Neither can be done at present using the composite form, without some ugly parsing from a previous exec action.

The proposal is to replace `installation.name` with the bundle name (stripped of namespace and /) and add `installation.namespace` to carry the namespace. This seems more consistent with how name and namespace are handled just about everywhere else.

Note: the CNAB_INSTALLATION_NAME env variable will not change: only the template variables derived from it. If you want the composite form, you can use either `{{ env.CNAB_INSTALLATION_NAME }}` or `{{ installation.namespace }}/{{ installation.name }}`.

An example of an action definition that might use these new variables:
```yaml
k8saction:
  - kubernetes:
      description: "Create some k8s resources"
      namespace: "{{ installation.namespace }}"
      manifests:
        - my-manifests
      wait: true
      outputs:
        - name: SVC_IP_ADDRESS
          namespace: "{{ installation.namespace }}"
          resourceType: service
          resourceName: "svc-{{ installation.name }}"
          jsonPath: "{.spec.clusterIP}"
```

**Additional Context**
I believe this only affects the porter Manifest.

**Risks/Concerns**
The proposal redefines `installation.name`, which may break some existing bundles. The alternative is to keep that as is (composite form) and create a new variable. But I feel that would be more confusing in the long run.

Do we need to vet this proposal to the community as a breaking change?

**Implementation details**
Pretty trivial and isolated changes.

# Checklist
- [ ] An announcement of this proposal has been sent to the Porter mailing list: https://porter.sh/mailing-list
- [ ] This proposal has remained open for at least one week, to allow time for community feedback.

**Comment by Jemgoss** [comment]
I've updated the title to describe the issue more generally.

A better solution has been proposed by @carolynvs to provide `PORTER_INSTALLATION_NAME` and `PORTER_INSTALLATION_NAMESPACE` environment variables. That way, we no longer change existing template vars and these values are available directly to action scripts without having to pass them in from the porter manifest.

**Comment by carolynvs** [comment]
Implemented in #2001
